agenta-web/cypress/e2e/ab-testing-evaluation.cy.ts

```diff
@@ -93,6 +96,34 @@ describe("A/B Testing Evaluation workflow", () => {
                 '[data-cy="evaluation-vote-panel-comparison-both-good-vote-button-button"]',
             ).click()
         })
+
+        it("Should successfully delete the evalueation", () => {
+            cy.url().should("include", "/annotations/human_a_b_testing")
+            cy.get('[type="checkbox"]').eq(1).check().should("be.checked")
+            cy.get('[data-cy="annotation-delete-button"]').click()
+            cy.get('[data-cy="annotation-table"] > .ant-table-tbody').should("not.exist")
+        })
+    })
+
+    context("When deleting the app variant", () => {
```

Comment:
this is not the right location ?? why the test for deleting a variant is in the evaluation suite ?

Reply:
During the A/B evaluation test, we are creating a new evaluation variant. After completing the evaluation, we test the deletion of the variant.

Reply:
@ashrafchowdury Let’s have it as an individual test case and not merged with the A/B evaluation test

Reply:
ok